content/blog/intro-to-gamedev/index.md

```diff
@@ -0,0 +1,560 @@
+- `VerticalSpeed`: Gravity accelerates our character downward, which in turn means it increases the vertical speed linearly (i.e: a specific amount per frame/time unit).
+
+### Updating the world with `delta`
+
+As mentioned before, the `delta` parameter that we are getting from the ticker is and passing on to our `Update()` method contains, in miliseconds, how much time passed between the previous frame and this one. Why is that value useful to us? Well, if we don't know how much time passed between one frame and the next one, we can't proportionally simulate the world. Usually, if we fix the framerate, we can simulate for a specific amount of frametime (so, if we want our game to run at 60 FPS, we would divide 1000 ms by 60 frames, which gives us 16.6 ms/frame), but in reality we can't rely on that being the real time that passed between two frames (there is a lot of variance introduced by the OS, the browser, the user, etc.).
+
+This way, if we want a value to increase by a rate of `X per second` depending on how much time passed, we need to adjust for how much time passed in the frame. If half a second passed between a frame and the next one, the amount by which we increase would be equal to `0.5 seconds` times `X per second`. This is the pattern that we will use to increment values that are tied to time (for example: user score, player speed, etc.).
+
+Keeping all this in mind, you might already get the sense of what the update code is going to look like, but to spell it out: If our character not airborne and the player pressed space, let's set the VerticalSpeed to a negative value so our little character goes up. If the character is airborne, we need to accelerate it downward by adding up a constant value to the vertical speed. To make sure he doesn't go downward eternally, let's check against a `GroundPosition` (defined on `GameApp` since it will depend on the view's height) so that we know when we are not airborne anymore (and when we can set the speed to 0). Finally, let's make our character move by changing the sprite's Y position:
+
+```ts
+public Update(delta: number)
+    {
+        if (this.sprite.y >= GameApp.GroundPosition)
+        {
+            // if downward acceleration brought us to the ground, stop and set airborne to false
+            this.sprite.y = GameApp.GroundPosition;
+            this.verticalSpeed = 0;
+            this.airborne = false;
+        }
+
+        if (this.airborne)
+        {
+            // if we are in the air, accelerate downward by increasing the velocity by a constant value
+            this.verticalSpeed += delta* 1/3;
+        }
+
+        if (GameApp.PressedSpace && !this.airborne)
+        {
+            // jump!
+            this.airborne = true;
+            this.verticalSpeed = -5;
+        }
+
+        // remember the delta update!
+        // the position will change in accordance to how much time passed and the character's speed
```

Comment:
super long lines here to (I wont comment every block, but take this comment into consideration for every code block)